<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Servizio</class>
 <widget class="QWidget" name="Servizio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dipendente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(69, 162, 0);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelNomeCognome">
          <property name="font">
           <font>
            <pointsize>30</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>NOME DIPENDENTE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rimuoviProfilo">
          <property name="styleSheet">
           <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);labella</string>
          </property>
          <property name="text">
           <string>Resetta account</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Codice fiscale:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="labelCF"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Indirizzo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="labelIndirizzo"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="labelEmail"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Telefono:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="labelTelefono"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);labella</string>
          </property>
          <property name="text">
           <string>Salva modifiche</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
